problemsets-mat223/modules/module14-exercises.tex

```diff
@@ -1,17 +1,42 @@
 \begin{exercises}
 	\begin{problist}
-		\prob  Example problem with parts
-		\begin{enumerate}
-			\item   The first part
-			\item   The second part
-		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
-			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
-			\end{enumerate}
-		\end{solution}
-
+		\prob Let $\mathcal T:\R^2\to\R^2$ be defined by $\mathcal T\mat{x\\y}=\matc{3x-y\\x-\tfrac{1}{4}y}$.
+	Find the volume of $\mathcal T(C_2)$.
+	
+	    \prob Let $\mathcal S:\R^3\to\R^3$ be defined by $\mathcal S\mat{x\\y\\z}=\matc{2x+y+z\\x-\tfrac{1}{2}y\\z}$.
+	Find the volume of $\mathcal S(C_3)$.
+	    
+	    \prob Let $\mathcal T:\R^2\to\R^2$ be defined by $\mathcal T\mat{x\\y}=\matc{x+2y\\-x-y}$.
+	    \begin{enumerate}
+			\item   Draw a picture of the transformation $\mathcal T$ to determine its orientation.
```

Comment:
What do you mean by draw a picture? You mean, "draw E and T(E) and then determine whether T is orientation preserving or orientation reversing", I presume?